docs/configuration/authentication/transports/cookie.md

```diff
@@ -24,7 +24,7 @@ As you can see, instantiation is quite simple. It accepts the following argument
 
 This method will return a response with a valid `set-cookie` header upon successful login:
 
-!!! success "`200 OK`"
+!!! success "`204 OK`"
```

Comment:
To be fully correct, it should be `204 No Content`. Could you change it? 🙂

Reply:
Oh yea definitely. Thanks for the quick suggestion 👍.